vm/src/obj/objlist.rs

```diff
@@ -96,6 +97,30 @@ impl PyList {
 
         start..stop
     }
+
+    pub fn get_byte_inner(&self, vm: &VirtualMachine) -> PyResult<objbyteinner::PyByteInner> {
+        let mut elements = Vec::<u8>::new();
```

Comment:
how about `Vec::<u8>::with_capacity(self.elements.borrow().len())`?

Reply:
Thank you for reviewing. Also there is implemented method `get_len`. in PyList. I will change to `Vec::<u8>::with_capacity(self.get_len())`